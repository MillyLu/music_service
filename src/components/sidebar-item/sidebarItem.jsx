import { NavLink } from "react-router-dom";
import * as Styled from './styles'

export function SidebarItem(props) {

    return(
        <Styled.SidebarItem>
            <NavLink className="App-link" to={props.href}>
                <Styled.SidebarImage src={props.image} alt='day"s playlist' />
            </NavLink>
        </Styled.SidebarItem>
    )
}
