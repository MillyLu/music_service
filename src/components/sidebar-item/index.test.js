/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { SidebarItem } from './sidebarItem';



describe('<SidebarItem/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<SidebarItem />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });