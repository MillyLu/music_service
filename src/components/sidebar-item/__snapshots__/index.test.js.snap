// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarItem/> should render successfully and match snapshot 1`] = `
<styled.div>
  <NavLink
    className="App-link"
  >
    <styled.img
      alt="day\\"s playlist"
    />
  </NavLink>
</styled.div>
`;
